app/services/patient_breakdown_service.rb

```diff
@@ -0,0 +1,43 @@
+class PatientBreakdownService
+  CACHE_VERSION = 1
+
+  def self.call(*args)
+    new(*args).call
+  end
+
+  def initialize(region:)
+    @region = region
+    @facilities = region.facilities
+  end
+
+  attr_reader :region
+  attr_reader :facilities
+
+  def call
```

Comment:
Pretty sure we need to handle other possible `@period`s here, right?

Reply:
Hm, now I'm confused:

* the actual details page does NOT have a date dropdown.
* the details page in the mockup does.

guess we should chat w/ @claudiovallejo about it.

Reply:
We decided on showing only the current period's data for the first iteration, without a datepicker. Once this is out, we can figure out a way to switch periods for the LTFU graph and figure out how to show the pie chart for previous periods (or if it is possible to do it).

Reply:
Sounds good.